docs/guides/hooks-list.rst

```diff
@@ -208,6 +211,14 @@ Ajax
 	Plugins can alter the output, forward URL, system messages, and errors. For the ``elgg/Ajax`` AMD module,
 	use the ``ajax_response`` hook documented above.
 
+**response, view:<view_name>**
+    Filter an instance of ``\Elgg\Http\ResponseBuilder`` before it is sent to the client
+    Applies to request to ``/ajax/view/<view_name>``
```

Comment:
Yes, response hook is called before anything else. It receives an instance of `Elgg\Http\ResponseBuilder`, so plugins can set the content, status code etc.